apps/gallery/tests/test_apis.py

```diff
@@ -205,3 +209,38 @@ def save_file_with_api(self, kwargs={}):
         return response.data['id']
 
     # NOTE: Test for files
+
+
+class PrivateAttachmentFileViewTest(TestCase):
+    def setUp(self):
+        # Create a test user
+        self.user = User.objects.create_user(username='testuser', password='testpassword')
```

Comment:
3 types of users.
- Normal
- Member (Non-confidentiality)
- Member (confidentiality)